lib/disc.rb

```diff
@@ -1,12 +1,18 @@
 require 'date'
 require 'disque'
 require 'msgpack'
+require 'connection_pool'
 
 require_relative 'disc/version'
 
 class Disc
-  attr_reader :disque,
-    :disque_timeout
+  def self.pool
+    @pool ||= begin
+      concurrency = Integer(ENV.fetch('DISC_CONCURRENCY', 25))
+      tolerance = Integer(ENV.fetch('DISC_TOLERANCE', disque_timeout / 1000.0))
```

Comment:
Decent defaults are all we can hope for, if a reason to change it presents itself then we'll reevaluate, for now that's good enough :)